client/reader/user-stream/views/lists.tsx

```diff
@@ -1,11 +1,23 @@
+import { formatListBullets, Icon } from '@wordpress/icons';
 import { useTranslate } from 'i18n-calypso';
+import EmptyContent from 'calypso/components/empty-content';
 
-const UserLists = () => {
+interface UserListsProps {
+	children?: React.ReactNode;
```

Comment:
1. Can we remove `?` here as we will always pass the children.
2. Nit: `children` is an ambiguos name so can we add a comment for better understanding?

Reply:
How do you feel about renaming `children` to `headerContent`? Something like this: e428ecc6cdc56a9074f09338626f92c2a62a22a2

Reply:
It's better, we can also move the header to its own component which will make both components simple.

Reply:
Something like this? 549f0155ed7f408fa8437b067ff031507a22b33f

Reply:
I like it! Implemented in 63ff7af089ca009cc73445e768a113e151643d2e

Reply:
It looks great, thanks for the implementation 🙂